en/docs/choreo-samples/quick-deploy-a-sample.md

```diff
@@ -23,11 +23,15 @@ To quickly deploy a sample, follow these steps:
 1. Sign in to the [Choreo Console](https://console.choreo.dev/).
 2. On the Choreo Console header, go to the **Project** list and select your project. This opens the project **Overview** page.
 3. On the project **Overview** page, go to the **Get Started with a Sample** section. You will see a selection of samples.
-4. To see the complete collection of samples, click **View All Samples**.
+4. To view the complete collection of samples, click **View All Samples**.
 5. Hover over a sample of your choice and click **Quick Deploy**. This creates the component and deploys it automatically.
 
-Once you quick deploy a sample, you can follow one of the following post-deployment actions given below depending on the component type:
+## Post-deployment actions
 
-- **For services**: Go to the **Test Console** to test the deployed service.  
-- **For web applications**: Open the deployed web application using the provided URL.  
-- **For manual tasks and scheduled tasks:** Go to the **Execute** page and run the task. For scheduled tasks, you can also find the cron schedule on the **Overview** page.
+Once you quick deploy a sample, follow one of the post-deployment actions below based on the component type:
+
+- **For Services**: Go to the **Test Console** to test the deployed service.
```

Comment:
Since we follow the Google style guide capitalization guidelines, we avoid using title caps for lists and such. https://developers.google.com/style/capitalization#capitalization-in-lists